tutorials/cloudsdk-integration-unit-test/cloudsdk-integration-unit-test.md

```diff
@@ -15,8 +15,6 @@ primary_tag: products>sap-cloud-sdk
 This tutorial assumes you are already familiar with the basics of the SAP Cloud SDK and project "Piper", e.g. because you already completed the mission [Create a Cloud Foundry App Using SAP Cloud SDK](mission.cloudsdk-cf-app) and the tutorial [Set Up Continuous Integration and Delivery for SAP Cloud SDK](cloudsdk-ci-cd). To follow this tutorial please download and extract an [example project](https://github.com/SAP/cloud-s4-sdk-book/archive/tutorials/testing-start.zip) based on the SAP Cloud SDK archetype.
 
 
-
-## Intro
 > ## We migrate tutorials to our [documentation](https://sap.github.io/cloud-sdk/)
```

Comment:
Do not header formatting to emphasize text. \*\*We migrate...\*\* would be correct in this case.